README.md

```diff
@@ -208,6 +213,183 @@ try {
 
 <a name="help"/>
```

Comment:
This link target is at wrong location. By the way, why are html links used instead markdown links? Sure, github supports them but users may open this file in other editors.

Reply:
fixed location. link is already in markdown style - anchor isn't it. is there markdown syntax for anchors?